src/modals/ImageModal/index.tsx

```diff
@@ -0,0 +1,33 @@
+import {createPortal} from 'react-dom';
+import {SFC} from 'types';
+import {useState} from 'react';
+import * as S from './Styles';
+import DefaultAvatar from '../../assets/default-avatar.svg';
+
+export interface ImageModalProps {
+  close(): void;
+  image: string | null;
+}
+
+const ImageModal: SFC<ImageModalProps> = ({image, close}) => {
+  const [preview, setPreview] = useState<string | null>(image ? image : DefaultAvatar);
```

Comment:
I don't think we should show a defaultAvatar whenever image is null.
DefaultAvatar will not make sense for images other than profile pictures.
Since we are making a generic component that will be used at other places too.

+ Ideally, the image src should should not be null

Reply:
You are right. I will make it string only